### DISCUSSION TITLE
Ability to not do a re-deploy when I need change env variables?

### DISCUSSION THREAD

**Discussion by karanganesan** [vercel/community#1513] [Idea]
It would be good to see this ability.
I random need to add commits once I change envs just to redeploy.

**Comment by karanganesan** [comment]
cc @leerob @rauchg

**Comment by amyegan** [comment]
Thanks for reaching out, Karan! [Edge Config](https://vercel.com/docs/concepts/edge-network/edge-config#) was designed to address the need for quick config changes without a site redeploy. It's good for things like feature flags, maintaining dynamic redirects, and updating custom request blocking rules, without a redeploy.

Would this work for your project?
